corehq/apps/reports/standard/deployments.py

```diff
@@ -487,6 +488,40 @@ def _fmt_timestamp(timestamp):
         result[0][1] = table
         return result
 
+    def get_location_column(self, user):
+        assigned_loc_ids = user.get('assigned_location_ids')
+        if not assigned_loc_ids:
+            return '---'
+        primary_loc_id = user.get('location_id')
+        return self._get_formatted_assigned_location_names(primary_loc_id, assigned_loc_ids)
```

Comment:
nit: you could make it simpler by passing the user to the function itself.
Fetching assigned_loc_ids and primary_loc_id isn't really needed in this function
```
def get_location_column(self, user):
    if not user.get('assigned_location_ids'):
        return '---'
    return self._get_formatted_assigned_location_names(user)
```

Reply:
[561afac](https://github.com/dimagi/commcare-hq/pull/34744/commits/561afac05c2a643648b1633077479634f74748bb)